From ffdda693484c186439610635adf00a32a92c0d8f Mon Sep 17 00:00:00 2001
From: "Frederic Pierret (Epitre)" <frederic.epitre@orange.fr>
Date: Tue, 19 Sep 2017 14:48:13 +0200
Subject: [PATCH 26/31] anaconda: lock root account by default

---
 pyanaconda/kickstart.py | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/pyanaconda/kickstart.py b/pyanaconda/kickstart.py
index fdd3f2a..bcb40cf 100644
--- a/pyanaconda/kickstart.py
+++ b/pyanaconda/kickstart.py
@@ -1680,6 +1680,11 @@ class ReqPart(commands.reqpart.F23_ReqPart):
         do_reqpart(storage, reqs)
 
 class RootPw(commands.rootpw.F18_RootPw):
+    def __init__(self, writePriority=100, *args, **kwargs):
+        if 'lock' not in kwargs:
+            kwargs['lock'] = True
+        super(RootPw, self).__init__(writePriority=writePriority, *args, **kwargs)
+	
     def execute(self, storage, ksdata, instClass, users):
         if flags.automatedInstall and not self.password and not self.seen:
             # Lock the root password if during an installation with kickstart
-- 
2.9.5

